src/logic/map_objects/world/terrain_description.h

```diff
@@ -55,7 +55,10 @@ class TerrainDescription {
 		const Image* icon;
 	};
 
-	TerrainDescription(const LuaTable& table, Descriptions& descriptions);
+	static constexpr uint16_t kMaxDitherLayerDisambiguator = 100;
```

Comment:
This deserves some commet:
// To avoid conflict with ditherlayers form mutiple pluigns we us this to ... 
// This should be enough for about 650 Plugins? (max uint16_t / 100)